src/client/src/components/form/formUtils.ts

```diff
@@ -58,6 +49,50 @@ export const getFieldBorderColor = (isReadOnly: boolean) => {
   };
 };
 
+/**
+ * Formats a number with scientific notation if the number is less than 0.001 and has more than 3 decimal places.
+ * @param {number} value The number to format.
+ * @returns The formatted number.
+ */
+
+const formatWithScientificNotation = (value: number) => {
+  const fullExpontential = value.toExponential();
+  const exponentialFractionDigits = getDecimalsFromNumericString(fullExpontential.split("e")[0]);
+  return value.toExponential(Math.min(exponentialFractionDigits, 3)).replace("e", " x 10");
+};
+
+/**
+ * Formats a number with thousands separators.
+ * @param {number} minDecimals The minimum number of decimal places to display.
+ * @param {number} value The number to format.
+ * @returns The formatted number.
+ */
+const formatWithThousandsSeparator = (minDecimals: number, value: number) => {
+  // Format number using de-CH
+  const formatted = new Intl.NumberFormat("de-CH", {
+    useGrouping: true,
+    minimumFractionDigits: minDecimals,
+    maximumFractionDigits: Math.max(3, minDecimals),
+  }).format(value);
+  // Ensure thousand separators are always a standard single quote (')
+  return formatted.replace(/’/g, "'");
+};
+
+/**
+ * Formats a number with thousands separators and a minimum number of decimal places.
+ * If the number is less than 0.001 and has more than 3 decimal places, the number is formatted with scientific notation.
+ * @param {number} value The number to format.
+ * @param {number} minDecimals The minimum number of decimal places to display (defaults to 0).
+ * @returns The formatted number.
+ */
+export const formatNumberForDisplay = (value: number | null, minDecimals = 0): string => {
+  if (value == null) return "-";
+  if (value < 0.001 && getDecimalsFromNumericString(value.toString()) > 3) {
```

Comment:
Q: `value.toString()` kann einen string in scientific notation generieren. z.B. `1e-7.toString()` gibt "1e-7". Mit diesem string kann aber `getDecimalsFromNumericString` nicht umgehen. Müssen wir mit so kleinen Zahlen umgehen können?

Reply:
Ja ich denke wir sollten mit so kleinen Zahlen umgehen können. Meine Testzahlen waren wohl nicht klein genug @domi-b @patrickackermann  kann jemand von euch übernehmen ich muss jetzt leider los